sql/firefox_accounts/fxa_amplitude_export_v1/view.sql

```diff
@@ -0,0 +1,125 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_accounts.fxa_amplitude_export_v1`
```

Comment:
Either this view should be "hidden" underneath `firefox_accounts_derived` or it should not have a version suffix.